### PR TITLE
Add a 'cfa' base path to user admin URL so the website works with nginx

### PR DESCRIPTION
**Pull request by maxime-poulain**
In production the redirection on cfa will add a /cfa base path this add the necessary things to make the site works.

### CODE REVIEW

**Comment by Victor-van-Duijnen** [comment]
Why is it tagged as bug? It's not a bug....

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Shared/_AdminLayout.cshtml [hunk 1]
```diff
@@ -36,7 +35,7 @@
                 <div class="c-toolbar__left">
                     <div class="c-toolbar__item">
                         <div class="c-brand c-brand--xsmall">
-                            <a href="/">
+                            <a href="https://account.ubik.be/cfa.aspx">
```
Comment (Victor-van-Duijnen) [inline src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Shared/_AdminLayout.cshtml:38]: I should be injected in options params. I opened another PR for it

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Program.cs [hunk 2]
```diff
@@ -71,10 +70,11 @@
     app.UseHttpsRedirection();
 }
 
-app.UseStaticFiles();
-
+app.UsePathBase(new PathString("/cfa"));
 app.UseRouting();
```
Comment (Victor-van-Duijnen) [inline src/UserAdmin/src/Smart.FA.Catalog.Web/Program.cs:74]:
```suggestion
app.UsePathBase(new PathString("/cfa"));

app.UseRouting();
```

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Shared/_AdminLayout.cshtml (hunk 1 above)
Comment (maxime-poulain) [reply src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Shared/_AdminLayout.cshtml:38]: Done alrady the changes :/

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Program.cs (hunk 2 above)
Comment (maxime-poulain) [reply src/UserAdmin/src/Smart.FA.Catalog.Web/Program.cs:74]: No need 
https://github.com/dotnet/aspnetcore/blob/c85baf8db0c72ae8e68643029d514b2e737c9fae/src/Http/Http.Abstractions/src/PathString.cs#L460 :)

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Program.cs [hunk 3]
```diff
@@ -51,18 +51,18 @@
 
 app.UseForwardedHeaders();
 
-
 if (app.Environment.IsProduction())
 {
-    app.UseExceptionHandler("/Error");
+    app.UseExceptionHandler("/cfa/Error");
```
[the remark this answers is not in the document]
Comment (maxime-poulain) [reply src/UserAdmin/src/Smart.FA.Catalog.Web/Program.cs:56]: That one however is a path I've switched that to Routes.*

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Index.cshtml.cs [hunk 4]
```diff
@@ -9,16 +9,13 @@ namespace Smart.FA.Catalog.Web.Pages;
...
     {
         _logger = logger;
     }
 
     public IActionResult OnGet()
     {
-        Identity = HttpContext.User.Identity as CustomIdentity;
-
-        return Page();
+        return RedirectToPage("/Admin/Index");
```
[the remark this answers is not in the document]
Comment (maxime-poulain) [reply src/UserAdmin/src/Smart.FA.Catalog.Web/Pages/Index.cshtml.cs:19]: When using RedirectToPage, it requires to uses the physical path to the page, while using Redirect you can use the Routes constant class.

I have been thinking lately if we should go all the way down with routes or physical paths both got their pros & cons.
We could maintain also a PagePaths something class but I am not a huge fan of maintaining two ways.

This is a good topic to talk about during the upcoming retro.

**Review by LievenPraet — APPROVED** [review]

**Pull request merged**